db/migrate/20220912152913_remove_sticky_en_topics.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveStickyEnTopics < ActiveRecord::Migration[6.1]
+  def up
+    Topic.where(id: [210_000, 220_000, 230_000, 240_000, 250_000]).delete_all
```

Comment:
тут и ниже `.destroy_all`-  всесте с топиками должны удалиться все связанные ассоциации
```suggestion
    Topic.where(id: [210_000, 220_000, 230_000, 240_000, 250_000]).destroy_all
```